lambdas/split/index.js

```diff
@@ -130,9 +141,9 @@ module.exports.handler = function (event, context, cb) {
 
 local.localRun(() => {
   const payload = {
-    satellite: 'landsat',
+    satellite: 'sentinel',
```

Comment:
@scisco I don't understand this change - the sat name is changed to Sentinel but the arn still is for Landsat step function.